### ISSUE TITLE
Request Presentation - Not showing data

### ISSUE THREAD

**Issue by premjayaraman** [OpenIotOrg/openiot#15]
The request presentation results in an unable to connect to SD&UM error.

The username is prem@csiro.au and pwd is prem

The property file is

scheduler.core.lsm.openiotMetaGraph=http://lsm.deri.ie/OpenIoT/demo/sensormeta#
scheduler.core.lsm.openiotDataGraph=http://lsm.deri.ie/OpenIoT/demo/sensordata#
scheduler.core.lsm.openiotFunctionalGraph=http://lsm.deri.ie/OpenIoT/guest/functional#
scheduler.core.lsm.access.username=openiot_guest
scheduler.core.lsm.access.password=openiot
# Service Delivery & Utility Manager (SD&UM) Properties

sdum.core.lsm.openiotFunctionalGraph=http://lsm.deri.ie/OpenIoT/guest/functional#
sdum.core.lsm.sparql.endpoint=http://lsm.deri.ie/sparql

Screendump of error
![request-presentation](https://f.cloud.github.com/assets/2141439/1433905/77fdb4b0-4104-11e3-84ce-5d6b3e2994b5.png)

**Comment by nkef** [comment]
Prem, I am not sure if this is the problem but I think Hoan has created the http://lsm.deri.ie/OpenIoT/guest/functionaldata# for the "scheduler.core.lsm.openiotFunctionalGraph" and the "sdum.core.lsm.openiotFunctionalGraph". Please replase that and try again

**Comment by nflynn** [comment]
When using the "Map" function from the "Sink" dropdown in the request defintion, the user can not enter negative co-ordinate values for the latitude or longitude boxes (i.e. Canberra = LATITUDE> -35.275489, LONGITUDE > 149.120717)

**Comment by nkef** [comment]
It seems there are hard-coded properties in the request definition module.

![request presentation hardcoded](https://f.cloud.github.com/assets/886559/1437193/50ec30f8-4169-11e3-9bdd-da002e6e9294.jpg)

Prem could you try to replace them with the global properties file (directions on how to use it can be found [here](https://github.com/OpenIotOrg/openiot/wiki/Openiot-Commons-Library#developer-1)) to configure the hard-coded variables?

**Comment by premjayaraman** [comment]
Nikos,

The bug with eclipse was again related to JAXB. JAXB was failing to unmarshall the data from SDUM. Hence, the request presentation was throwing an error.

The fix that has worked for me is to add a dependency in the POM file as below
        
        <dependency>
            <groupId>org.eclipse.persistence</groupId>
            <artifactId>org.eclipse.persistence.moxy</artifactId>
            <version>2.5.1</version>
        </dependency>

I tried you previous solution but it did not work. Resulted in the same error.

**Comment by stavrosp** [comment]
Hi Prem,
I am just sharing my experience with jaxb since I had the exact same problem when I tried to install the platform to 2 different pc's (using eclipse). The solution that you provided with the moxy dependency works great. 

After checking for some other possible solutions I found that If we update the jaxb dependencies so that a new version of jaxb is deployed for example:

<dependency>
    <groupId>javax.xml.bind</groupId>
    <artifactId>jaxb-api</artifactId>
    <version>2.2.6</version>
</dependency>
<dependency>
    <groupId>com.sun.xml.bind</groupId>
    <artifactId>jaxb-impl</artifactId>
    <version>2.2.6</version>
</dependency>

then everything works fine too.
